msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,168 @@ The following image shows the default view of the Adaptive Card when you have no
+
+  |Value  |Description  |
+  |---------|---------|
+  | `veryNarrow` | The card width on a meeting chat pane on desktop. |
+  | `narrow` | The card width on a mobile phone in portrait mode. |
+  | `standard` | The card width on a mobile phone in landscape mode, tablet in portrait mode, and chats in desktop. |
+  | `wide` | The card width on a tablet in landscape mode, and chats and channels in desktop. The card width on a chat pane on desktop can vary from `standard` to `wide`. |
+
+  
+
+  The value of `targetWidth` refers to the relative width of the card and allows more flexibility in adapting to different screen sizes. You can decide that certain elements are visible as per the width of the card you're designing. For example, when you design a responsive card, you can decide that certain elements are only visible when the width of the card is very narrow. To do that, you should set `targetWidth`:`veryNarrow` on that specific element.
+
+  You can add the `targetWidth` property to any element with a prefix to a valid value that is `atLeast` or `atMost`. When you specify the targetWidth value to a value with a prefix, the elements are displayed or hidden based on the card width.
+
+  The following examples explain how the elements can be made visible or hidden based on the card width:
+
+  * If you set `targetWidth` to `standard`, the element is only visible when the card width is standard.
+
+  * If you set `targetWidth` to `veryNarrow`, the element is only visible when the card width is very narrow.
+
+  * If you set `targetWidth` to `atLeast:narrow`, the element is only visible when the card width is either narrow, standard, or wide and not when the card width is very narrow.
+
+  * If you set `targetWidth` to `atMost:narrow`, the element is only visible when the card width is either very narrow or narrow and not when the card width is standard or wide. 
+
+The following images show an example of a non responsive and responsive Adaptive Cards:
+
+**Adaptive Card without `targetWidth` configuration**
+
+:::image type="content" source="../../assets/images/Cards/non-responsive-adaptive-card.png" alt-text="Screenshot shows how adaptive card renders without the targetWidth configuration.":::
+
+**Adaptive Card responsive layout with `targetWidth` configuration**
+
+:::image type="content" source="../../assets/images/Cards/adaptive-card-responsive-layout.png" alt-text="Screenshot shows how adaptive card renders when the targetWidth is configured to respective card widths.":::
+
+#### Sample Adaptive Card responsive layout
+
+To make a responsive Adaptive Card, your app must include the elements from the following code sample:
```

Comment:
For the example to be valuable to developers, we need to show a card designed without using `targetWidth` and show how it doesn't scale well across different card widths. Then we need to show how **that same card** can be updated to use `targetWidth` to make it responsive, and again show how it now looks good across different card widths.

Currently, this page uses two completely different cards to illustrate non-responsive vs responsive cards, and doesn't provide the card payloads. That isn't going to be very useful to developers.

Then an actual card payload example follows (below) but:
- That example isn't either of the cards above
- No screenshot are included to show how the use of `targetWidth` helps the card scale